src/main/resources/templates/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<h1>Strona główna</h1>
+<ul>
+    <li><a href="/lista?kategoria=wszystkie">Wyświetl wszystkie produkty</a></li>
+    <li><a href="/lista?kategoria=spozywcze">Wyświetl art. spożywcze</a></li>
+    <li><a href="/lista?kategoria=domowe">Wyświetl art. gosp. domowego</a></li>
+    <li><a href="/lista?kategoria=inne">Wyświetl inne</a></li>
```

Comment:
To jako nieobowiązkowy bonus, ale mógłbyś w tym miejscu i w select w formlularzu spróbować wykorzystać pętlę. W kontrolerze możesz zapisać w modelu tablicę ze wszystkimi kategoriami, tj. Category.values(), a tutaj po tej tablicy iterować. Z tego co widzę, to w kategorii masz zarówno tłumaczenie, jak i link, więc wszystko, czego potrzebujesz. Zaletą wykorzystania w tym miejscu pętli byłoby to, że jeżeli w przyszłości dodać nową kategorię, to w szablonie nie będziesz musiał robić żadnej zmiany.

Reply:
Na poczatku tak nawet miałem, ale nie wiedziałem co zrobić z kategorią wszystkie, która też mi się wyświetlała ale służy raczej do filtrowania.